.github/workflows/deploy.yml

```diff
@@ -18,25 +27,21 @@ jobs:
       contents: write
 
     steps:
-      - name: Download content_config
-        uses: actions/download-artifact@v4
+      - name: Checkout content
+        uses: actions/checkout@v4
         with:
-          name: ${{ inputs.config_artifact_name }}
-          path: ./content_config
-
+          repository: ${{ inputs.content_organization }}/${{ inputs.content_repository }}
+          ref: ${{ inputs.content_ref }}
+          path: ./${{ inputs.content_repository }}
       
-      - name: Set environment variable
+      - name: Set content path environment variable
+        run: echo "CONTENT_PATH=${PWD}/${{ inputs.content_repository }}" >> $GITHUB_ENV
```

Comment:
Liked also the way you orginized things in a such a way to be able set up manually the content repo we want to render via `CONTENT_PATH`.